README.md

```diff
@@ -43,10 +43,29 @@ If you omit `npm-token`, then packages will be prepared for publishing, but no p
 
 You can optionally send deployment announcements to Slack by providing a `slack-webhook-url` input:
 
+The absolute minimum configuration for this is:
+
+```yaml
+- uses: MetaMask/action-npm-publish@v2
+  with:
+    slack-webhook-url: ${{ secrets.SLACK_WEBHOOK_URL }}
+```
+
+We've now added the ability to customize the message posted in Slack and those optional inputs are as follows:
+
+- `icon-url`: Url to the avatar used for the bot in Slack. If not set this defaults to the avatar in this repository.
```

Comment:
There's an API section below this; maybe we want to move this list below? i.e. We could use the Usage section to explain various ways you can use this action, but then for reference purposes, go into more detail about what each option does in the API section.

Reply:
ah! good call. I'll switch things around a bit to be more consistent.